src/main/java/com/notably/model/suggestion/SuggestionItemImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.notably.model.suggestion;
+
+/**
+ * Implementation class for Suggestion Item.
+ */
+public class SuggestionItemImpl implements SuggestionItem {
+    private String displayText;
+    private Runnable action;
+
+    public SuggestionItemImpl(String displayText, Runnable action) {
+        this.displayText = displayText;
+        this.action = action;
+    }
+
+    @Override
+    public String getDisplayText() {
+        return displayText;
+    }
+
+    @Override
+    public Runnable run() {
+        return action;
```

Comment:
Oops, I think we missed this during the interfaces PR and discussions before that. If you're going to return the runnable to the user, shouldn't this be called `getAction()`?

I feel that the method name `run()` sounds more like asking the method to run the `Runnable` directly. You can also do that by changing the method signature to `public void run()` and calling `action.run()` in the method body